lighthouse-cli/bin.ts

```diff
@@ -143,6 +147,10 @@ if (!cliFlags.outputPath && cliFlags['output-path']) {
   cliFlags.outputPath = cliFlags['output-path'];
 }
 
+if (!cliFlags.maxWaitForLoad && cliFlags['max-wait-for-load']) {
```

Comment:
how would this be possible? AFAIK, yargs creates the former and makes sure they match at this point.